packages/suite/src/views/settings/debug/CoinjoinApi.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { COINJOIN_NETWORKS } from '@suite/services/coinjoin/config';
+import { ActionColumn, ActionSelect, SectionItem, TextColumn } from '@suite-components/Settings';
+import * as suiteActions from '@suite-actions/suiteActions';
+import { useSelector, useActions } from '@suite-hooks';
+import type { CoinjoinServerEnvironment } from '@suite-common/wallet-types';
+
+const StyledActionSelect = styled(ActionSelect)`
+    min-width: 256px;
+`;
+
+export const CoinjoinApi = () => {
+    const { setDebugMode } = useActions({
+        setDebugMode: suiteActions.setDebugMode,
+    });
+    const { debug } = useSelector(state => ({
+        debug: state.suite.settings.debug,
+    }));
```

Comment:
just a note for the future - we would like to stop using return objects in selectors as described here https://www.notion.so/satoshilabs/Redux-b0a4c3b7e71b499a8d592fb44cd794fe#4f4936ae00f84140bc433300ae2b7d88

```suggestion
    const debug = useSelector(state => state.suite.settings.debug);
```